src/pages/AddItem/index.jsx

```diff
@@ -0,0 +1,202 @@
+  align-items: center;
+`;
+
+const AddProduct = styled.div`
+  font-size: 20px;
+`;
+
+const AddButton = styled.button`
+  width: 74px;
+  height: 42px;
+  border-radius: 8px;
+  background-color: ${({ disabled }) => (disabled ? "#9ca3af" : "#3692FF")};
+  border: none;
+  color: #f3f4f6;
+  cursor: ${({ disabled }) => (disabled ? "default" : "pointer")};
+`;
+
+const ProductUpLoadWrapper = styled.div`
+  display: flex;
+  gap: 16px;
+  width: 100%;
+  height: auto;
+  flex-direction: column;
+`;
+
+const ProductImage = styled.div`
+  font-size: 16px;
+  color: #1f2937;
+`;
+
+const NameWrapper = styled.div`
+  width: 100%;
+`;
+
+const ProductName = styled.div`
+  font-size: 16px;
+  color: #1f2937;
+`;
+
+const InputName = styled.input`
+  width: 100%;
+  padding: 16px 24px;
+  border: none;
+  background-color: #f3f4f6;
+  border-radius: 12px;
+  box-sizing: border-box;
+`;
+
+const ProductDescriptionWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+`;
+
+const ProductDescription = styled.div`
+  font-size: 16px;
+  color: #1f2937;
+`;
+
+const ProductDescriptionInput = styled.input`
+  width: 100%;
+  height: 282px;
+  padding: 16px 24px;
+  border: none;
+  border-radius: 12px;
+  background-color: #f3f4f6;
+  box-sizing: border-box;
+`;
+
+const PriceWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+`;
+
+const ProductPrice = styled.div`
+  font-size: 18px;
+  color: #1f2937;
+`;
+
+const ProductPriceInput = styled.input`
+  width: 100%;
+  padding: 16px 24px;
+  border: none;
+  border-radius: 12px;
+  background-color: #f3f4f6;
+  box-sizing: border-box;
+`;
+
+const TagWrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+`;
+
+const Tag = styled.div`
+  font-size: 18px;
+  color: #1f2937;
+`;
+
+const TagInput = styled.input`
+  width: 100%;
+  padding: 16px 24px;
+  border: none;
+  border-radius: 12px;
+  background-color: #f3f4f6;
+  box-sizing: border-box;
+`;
+
+const AddItem = () => {
+  const [name, setName] = useState("");
+  const [description, setDescription] = useState("");
+  const [price, setPrice] = useState("");
+  const [tag, setTag] = useState("");
+
+  const isFormValid = name && description && price && tag;
+
+  const handlePriceChange = (e) => {
+    const onlyNumbers = e.target.value.replace(/\D/g, "");
+    setPrice(onlyNumbers);
+  };
+
+  return (
+    <>
+      <Header />  
+      <Container>
+        <AddWrapper>
+          <AddProduct>상품 등록하기</AddProduct>
+          <AddButton disabled={!isFormValid}>등록</AddButton>
+        </AddWrapper>
+
+        <ProductUpLoadWrapper>
+          <ProductImage>상품 이미지</ProductImage>
+          <ImageUpload />
+        </ProductUpLoadWrapper>
+
+        <NameWrapper>
+          <ProductName>상품명</ProductName>
+          <InputName
+            type="text"
+            placeholder="상품명을 입력해주세요"
+            value={name}
+            onChange={(e) => setName(e.target.value)}
```

Comment:
💊 제안
지금처럼 인풋 값을 useState로 제어하면 사용자가 입력할 때마다 컴포넌트가 리렌더링됩니다.
지금 코드상에서는 인풋 값을 바꿔주는 것이 아니라 제어 컴포넌트로 관리할 필요가 없을 것 같아요.
이를 비제어 컴포넌트로 바꾸셔서 불필요한 리렌더링을 줄이시는 것을 추천드립니다.
```suggestion
            onChange={(e) => setName(e.target.value)}
```